packages/console/src/components/Uploader/FileUploader/index.tsx

```diff
@@ -43,35 +43,43 @@ function FileUploader({
   const api = useApi();
 
   const onDrop = useCallback(
-    async (acceptedFiles: File[]) => {
+    async (acceptedFiles: File[] = [], fileRejection: FileRejection[] = []) => {
       setUploadError(undefined);
 
-      const selectedFile = acceptedFiles[0];
+      const rejectedFile = fileRejection[0]?.file;
+
+      if (rejectedFile) {
+        /**
+         * Note:
+         * We need to display this invalid file type error, since the user can select an invalid file type by modifying the file input dialog settings.
+         */
+        if (!allowedMimeTypes.map(String).includes(rejectedFile.type)) {
+          setUploadError(
+            t('components.uploader.error_file_type', {
+              extensions: convertToFileExtensionArray(allowedMimeTypes),
+            })
+          );
+        }
 
-      if (!selectedFile) {
         return;
       }
 
-      if (!allowedMimeTypes.map(String).includes(selectedFile.type)) {
-        setUploadError(
-          t('components.uploader.error_file_type', {
-            extensions: convertToFileExtensionArray(allowedMimeTypes),
-          })
-        );
+      const acceptedFile = acceptedFiles[0];
 
+      if (!acceptedFile) {
         return;
       }
```

Comment:
Do we have total count check for the acceptedFiles? What if the user drag-and-drops two files and the code here seems to be only checking the first one

Reply:
We guard the file count in the `input` tag by setting `multiple` to `false`. so we will get only 1 file in the `acceptedFiles`. but It's good to add an error when we receive more than 1 file.

Reply:
ok